features/polyExplorer/src/components/clusterStory/clusterStory.jsx

```diff
@@ -43,7 +43,10 @@ const ClusterStory = ({
     };
 
     return (
-        <Screen className={`cluster-story ${className}`} light={true}>
+        <Screen
+            className={`cluster-story ${className} poly-theme-light`}
+            light={true}
```

Comment:
Wouldn't it be nicer to have a `theme` attribute here that sets the `poly-theme-light` class accordingly? Seems like we wouldn't need the `light` attribute anymore then.

(Also, with the current state of the code, wouldn't it need to be `poly-theme.light` instead of `poly-theme-light`?)